### DISCUSSION TITLE
Is there a way to supress the logs from bigquery schema generator?

### DISCUSSION THREAD

**Discussion by kabilan-dt** [bxparks/bigquery-schema-generator#96] [Q&A]
![image](https://github.com/bxparks/bigquery-schema-generator/assets/93244451/f55d55f9-3e82-4f87-873f-1c1e51547312)

Currently when we use schema generator, it prints all these logs. Is there a way tto disable these?

**Comment by bxparks** [comment]
`generate_schema.py` uses the normal [Python logging](https://docs.python.org/3/howto/logging.html) mechanism. Your log output contains the line number where those log outputs are generated (e.g. `generate_schema.py: 195`). You have access to the source code. You should be able to customize it to whatever you need it to do. But if you are really really really really pressed for time, maybe the [--debug_interval](https://github.com/bxparks/bigquery-schema-generator#DebuggingInterval) flag is sufficient.
